pkitesting/src/main/java/io/netty/pkitesting/CertificateBuilder.java

```diff
@@ -921,9 +947,6 @@ public enum Algorithm {
             this.signatureType = signatureType;
         }
 
-        static final AlgorithmParameterSpec UNSUPPORTED = new AlgorithmParameterSpec() {
-        };
```

Comment:
Turns out enum variants are initialized before any static fields on the enum, so this is moved to the `CertificateBuilder` class, to avoid `namedParameterSpec` returning `null` from the uninitialized field.